### PR TITLE
Jinja2ized quickstart

### PR DESCRIPTION
**Pull request by tk0miya**
To be quickstart simple, I refactored it.
I seperated the inline Makefile, make.bat, master_doc and conf.py to template files.

### CODE REVIEW

File: sphinx/quickstart/Makefile.new_t [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Minimal makefile for Sphinx documentation
```
Comment (shimizukawa) [inline sphinx/quickstart/Makefile.new_t]: I think it is better if template files are in `sphinx/templates` directory with other template files if needed.

**Comment by shimizukawa** [comment]
LGTM with nits. Great works!

File: sphinx/quickstart/Makefile.new_t (hunk 1 above)
Comment (shimizukawa) [inline sphinx/quickstart/Makefile.new_t]: ... or `sphinx/quickstart/templates/` is also nice ;)

Comment (tk0miya) [inline sphinx/quickstart/Makefile.new_t]: Moved to `sphinx/templates/quickstart`.

**Comment by lehmannro** [comment]
I think at some point we discussed making sphinx-quickstart incremental (and, then, a lot slimmer.)  I don't want to see this effort go to waste, but does that move us in that direction?  I think for incremental quickstarts we would probably want to get rid of Jinja templates again.

**Comment by tk0miya** [comment]
Sounds good. Of course yes. Please let me know your idea.

**Comment by tk0miya** [comment]
any updates? if not yet, I want to discuss after merging this.
I think this PR is simple refactoring. so merging this does not prevent the discussion.

**Comment by tk0miya** [comment]
Merged.  Please file new issue to discussion if you want.
Thanks,

**Pull request merged**